pkg/scheduler/apis/config/scheme/scheme_test.go

```diff
@@ -291,6 +297,73 @@ profiles:
 								BindTimeoutSeconds: 600,
 							},
 						},
+						{
+							Name: "PodTopologySpread",
+							Args: &config.PodTopologySpreadArgs{},
+						},
+					},
+				},
+			},
+		},
+		{
+			name: "empty PodTopologySpread, feature DefaultPodTopologySpread enabled",
+			data: []byte(`
+apiVersion: kubescheduler.config.k8s.io/v1beta1
+kind: KubeSchedulerConfiguration
+profiles:
+- pluginConfig:
+  - name: PodTopologySpread
+    args:
+      defaultConstraints:
```

Comment:
This works the same with:

```yaml
- name: PodTopologySpread
```

correct?

Reply:
It does, but I wanted to be more explicit